dpos/manager/dposmanager.go

```diff
@@ -550,11 +550,7 @@ func (d *DPOSManager) changeHeight() {
 }
 
 func (d *DPOSManager) clearEvidence(b *types.Block) {
```

Comment:
`clearEvidence` is a empty function now, it can be removed.

Reply:
Ok, fixed it